package.lisp

```diff
@@ -1877,8 +1877,10 @@
    ;; sheets
    #:always-repaint-background-mixin
    #:never-repaint-background-mixin
-
-   #:line-style-effective-thickness
+   #:background
+   #:foreground
+   ;; #:line-style-effective-thickness
```

Comment:
Is this intentional? The commit message doesn't mention it.

Reply:
Yes, I don't think this belongs in extensions (especially that it was not defined in all backends until now). We may think about readding it there /after/ we have working protocol in climb package.